docs/content/how-to/aws/troubleshooting.md

```diff
@@ -121,4 +121,26 @@ oc delete aws-ebs-csi-driver-controller aws-ebs-csi-driver-operator
 
 The operator will take a while to raise up again and eventually the driver controller will be deployed by the `aws-ebs-csi-driver-operator`.
 
-**Cause:** This issue probably come from objects that are deployed by the Operator, in this case the storage one, but the controller or the operator does not reconcile over them. If you delete the deployments you ensure that the operator recreates them from scratch.
\ No newline at end of file
+**Cause:** This issue probably come from objects that are deployed by the Operator, in this case the storage one, but the controller or the operator does not reconcile over them. If you delete the deployments you ensure that the operator recreates them from scratch.
```

Comment:
`...in this case the storage one...`, could we possibly be explicit about which exact operator this is? I think it's the `cluster-storage-operator`, right?